Sources/ApolloCodegenLib/Frontend/JavaScriptBridge.swift

```diff
@@ -94,9 +94,11 @@ extension JavaScriptObject: Equatable {
 class JavaScriptBridge {
   private var context: JSContext
   
-  // In JavaScript, classes are represented by constructor functions.
-  // We keep a bidirectional mapping between constructors and wrapper types so we can create wrappers
-  // for JavaScript objects and
+  // In JavaScript, classes are represented by constructor functions. We need access to these when checking
+  // the type of a received value in `wrap(_)` below.
+  // We keep a bidirectional mapping between constructors and wrapper types so we can both access the
+  // corresponding wrapper type, and perform an `instanceof` check based on the corresponding constructor
+  // for the expected wrapper type in case there isn't a direct match and we are receiving a subtype.
```

Comment:
This is definitely...more information. :)

This is still pretty dense, though. Remember that most folks reading this probably don't have extensive knowledge of JS, so this needs to be a lot less jargon-y.

Reply:
I don't think without this level of knowledge of JS you can really make changes to the bridge :) So I don't know who the audience for a more detailed explanation would be. This is definitely not something you'd have to deal with if you were just writing wrapper types, it's part of the implementation of the bridge.

Reply:
Hm. @hwillson does this make sense to you coming from JS land, then?

Reply:
@designatednerd Yep, looks good to me! 👍